lib/dams/controller_helper.rb

```diff
@@ -675,8 +675,12 @@ def audit( id = "unknown" )
     def referrer_controller( request )
       uri = URI(request.referrer || "")
       if uri.host == request.host
```

Comment:
I see, that makes sense in the test context.

My concern is the begin/rescue block, while functional, doesn't really tell
us what purpose it serves by simply reading the code. Would something like
the Rails.root work instead of the hardcoded '/dc'?

On Wed, Mar 25, 2015 at 4:10 PM lsitu <notifications@github.com> wrote:

> In lib/dams/controller_helper.rb
> <https://github.com/ucsdlib/damspas/pull/92#discussion_r27175659>:
>
> > @@ -675,8 +675,12 @@ def audit( id = "unknown" )
> >      def referrer_controller( request )
> >        uri = URI(request.referrer || "")
> >        if uri.host == request.host
>
> I tried it earlier but some tests failed. I think this is because /dc is
> environment specific and our test environment don't have the /dc url
> pattern, so it failed to retrieve the controller from the referrer url.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ucsdlib/damspas/pull/92/files#r27175659>.
>